ci/container_index/lib/state.py

```diff
@@ -0,0 +1,59 @@
+from os import path, mkdir, remove
```

Comment:
as I reviewed and remarked earlier, please also document at the start of every module.

Reply:
Added